default_payment_config.xml

```diff
@@ -29,13 +29,14 @@
         <wdoxidee_httppass>qD2wzQ_hrc!8</wdoxidee_httppass>
         <wdoxidee_descriptor>1</wdoxidee_descriptor>
         <wdoxidee_additional_info>1</wdoxidee_additional_info>
+        <initial_title>wd_heading_title_creditcard</initial_title>
     </payment>
     <payment>
       <oxid>wdpayolution-b2b</oxid>
       <oxactive>0</oxactive>
       <oxfromamount>10</oxfromamount>
       <oxtoamount>3500</oxtoamount>
-      <oxdesc id="wd_heading_title_payolution_b2b"/>
+      <oxdesc id="wd_heading_title_payolution_b2b_custom"/>
```

Comment:
Why does this need to be set as an attribute (id), whereas `initial_title` is set as a value? Those are both language keys, right?

Reply:
Yes, those are both the language keys, and initially they are the same. ```initial_title``` is added so that merchant can see what was the original payment method title, in case he changes ```desc``` value in the meanwhile.

```initial_title``` is shown on payment method configuration page in admin, under the logo.

Reply:
I used the same notation for adding metadata field ```initial_title``` like for all the other (metadata) fields. The choice of using "id" for ```desc``` was made in [this PR](https://github.com/wirecard/oxid-ee/pull/162).

Reply:
initial_title is translated via the normal translation function(the key is saved and provided to the function),
for oxdesc(id) the translation gets saved to the db (in OxidEeEvents) and can be changed by the merchant